internal/v1/handler_blueprints_test.go

```diff
@@ -494,19 +494,20 @@ func TestHandlers_DeleteBlueprint(t *testing.T) {
 	require.Len(t, result.Data, 0)
 
 	// We should not be able to update deleted blueprint
-	id5 := uuid.New()
-	err = dbase.UpdateBlueprint(ctx, id5, blueprintId, "000000", "newName", "desc2", json.RawMessage(`{"image_requests": [{"image_type": "aws"}, {"image_type": "gcp"}]}`))
+	err = dbase.UpdateBlueprint(ctx, blueprintId, blueprintId, "000000", "newName", "desc2", json.RawMessage(`{"image_requests": [{"image_type": "aws"}, {"image_type": "gcp"}]}`))
```

Comment:
Why are we changing this? I know it's useless to generate new UUID, but it might fail for other reasons when we do pass already used ID, so I'd rather keep only one possible failure, the one we test for.
Was there a reason other than safe some bytes?

Reply:
I also do not understand, if this is some irrelevant test improvement, I suggest to move it into a separate PR because this change should work without touching a single test.

Reply:
The other change is needed, we've used the same UUID to recreate blueprint with the same name, but that won't work anymore, we need new uuids as the current one will be already be there and we'd fail unique test

Reply:
This was unintentional, and you are correct. This does not belong in here.